pgxn/neon/libpagestore.c

```diff
@@ -722,6 +723,26 @@ get_socket_stats(int socketfd, int *sndbuf, int *recvbuf)
 #endif
 }
 
+/*
+ * Tries to get the local port of a socket. Sets 'port' to -1 on error.
+ */
+static void
+get_local_port(int socketfd, int *port)
+{
+#ifdef __linux__
```

Comment:
sockaddr_in and getsockname are in POSIX, do we need the guard macro?

Reply:
Ok, I removed them. Initially, I thought it's safer and we don't need this info on MacOS or other platforms anyway.